frontend/src/pages/profile/profile.jsx

```diff
@@ -15,10 +16,21 @@ class Profile extends React.Component {
         this.form = createRef();
         this.state = {
             user: this.props.user || null,
-            file: null
+            file: null,
```

Comment:
could you name these better so its more clear what the difference between the two is?